Resources/Private/JavaScript/Host/Containers/RightSideBar/Inspector/SingleInspectorEditor/index.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {$transform} from 'plow-js';
+
+import {CR, UI} from 'Host/Selectors/';
+import {Editors} from 'Components/index';
+import {actions} from 'Host/Redux/';
+
+const resolveEditor = (legacyEditorName) => {
+    // NOTE: this is the extension point for custom editors later.
+    switch (legacyEditorName) {
+        case 'TYPO3.Neos/Inspector/Editors/TextFieldEditor':
+            return Editors.TextField;
+        case 'TYPO3.Neos/Inspector/Editors/BooleanEditor':
+            return Editors.Boolean;
+        case 'TYPO3.Neos/Inspector/Editors/DateTimeEditor':
+            return Editors.DateTime;
+        case 'TYPO3.Neos/Inspector/Editors/SelectBoxEditor':
+            return Editors.SelectBox;
+        case 'TYPO3.Neos/Inspector/Editors/ImageEditor':
+            return Editors.Image;
+        case 'TYPO3.Neos/Inspector/Editors/TextAreaEditor':
+            return Editors.TextArea;
+        case 'Content/Inspector/Editors/NodeTypeEditor':
+            return Editors.NodeType;
+
+        default:
+            console.error(`ERROR: ${legacyEditorName} is not implemented`);
+            return null;
+    }
+};
+
+@connect($transform({
+    focusedNode: CR.Nodes.focusedSelector,
+    currentInspectorValue: UI.Inspector.currentValue
+}), {
+    writeValue: actions.UI.Inspector.writeValue
+})
+export default class SingleInspectorEditor extends Component {
```

Comment:
I could think about refactoring this one to a reusable component, so it could be used e.g. in the wizard. But I guess it's OK to leave it here for now, not hard to refactor it later.